### PR TITLE
Enable Github sourcelink

### PR DESCRIPTION
**Pull request by eddynaka**
https://docs.microsoft.com/dotnet/standard/library-guidance/sourcelink

### CODE REVIEW

**Comment by eddynaka** [comment]
Old package:
image

New Package:
image
image

File: src/Sarif/Sarif.csproj [hunk 1]
```diff
@@ -96,5 +97,9 @@
 
   <Import Project="ToDotNet\ToDotNet.targets" />
 
-  <ProjectExtensions><VisualStudio><UserProperties CodeGenHints_1json__JSONSchema="http://json.schemastore.org/asmdef" /></VisualStudio></ProjectExtensions>
+  <ProjectExtensions>
+    <VisualStudio>
```
Comment (michaelcfanning) [inline src/Sarif/Sarif.csproj:101]: >VisualStudio [](start = 5, length = 12)

This can be deleted from merge, I think.

Comment (eddynaka) [reply src/Sarif/Sarif.csproj]: just removed! thank you

---
In reply to: [520763919](https://github.com/microsoft/sarif-sdk/pull/2148#discussion_r520763919) [](ancestors = 520763919)

Comment (michaelcfanning) [reply src/Sarif/Sarif.csproj]: merge away!

---
In reply to: [520765577](https://github.com/microsoft/sarif-sdk/pull/2148#discussion_r520765577) [](ancestors = 520765577,520763919)

**Pull request merged**